omero/institution/getting-started.html

```diff
@@ -50,7 +50,7 @@ <h4 class="text-center">Groups and permissions system</h4>
                         <h4 class="text-center">Installing OMERO.server</h4>
                         <i class="fa fa-book fa-5x icon-green"></i>
                         <p class="card-caption">How to guides are provided for installing the server on a range of supported platforms and this page also links to information for getting your data repository, PostgreSQL database and OMERO.web deployment up and running.</p>
-                        <div class="text-center"><a class="button small" href="https://docs.openmicroscopy.org/omero/{{ site.omero.version }}/sysadmins/unix/index.html" target="_blank">Read the Guide</a></div>
+                        <div class="text-center"><a class="button small" href="https://docs.openmicroscopy.org/omero/{{ site.omero.version }}/sysadmins/unix/" target="_blank">Read the Guide</a></div>
```

Comment:
This goes to a directory listing. I would propose we point at `https://docs.openmicroscopy.org/omero/{{ site.omero.version }}/sysadmins/index.html` instead.